Process.c

```diff
@@ -350,7 +350,7 @@ void Process_makeCommandStr(Process* this, const Settings* settings) {
       return;
    }
 
-   int exeLen = strlen(this->procExe);
+   int exeLen = (int)MINIMUM(strlen(this->procExe), 65536);
    int exeBasenameOffset = this->procExeBasenameOffset;
    int exeBasenameLen = exeLen - exeBasenameOffset;
```

Comment:
Switch to `size_t` would be better (DiD).

Reply:
Just to mention I don't like arbitrary limit here. If this `procExe` was for display, we can cut the length to as short as `PATH_MAX`, or is there a case where `PATH_MAX` is not enough?